--- vpopmail-5.3.3/configure.in.orig	2004-01-13 23:17:09.000000000 +0100
+++ vpopmail-5.3.3/configure.in	2004-01-13 23:19:31.000000000 +0100
@@ -69,11 +69,6 @@
         fi
     ] )
 
-if test ! -f "$qmailnewu"
-then
-    AC_MSG_ERROR([Unable to find your qmail-newu file, specify --enable-qmail-newu=/full/path/to/qmail-newu])
-fi
-
 AC_DEFINE_UNQUOTED(QMAILNEWU,"$qmailnewu")
 
 qmailinject=""
@@ -92,10 +87,6 @@
             AC_MSG_ERROR([Unable to find your qmailinject file, specify --enable-qmail-inject.])
         fi
     ] )
-if test ! -f "$qmailinject"
-then
-    AC_MSG_ERROR([Unable to find your qmail-inject file, specify --enable-qmail-inject=/full/path/to/qmail-inject])
-fi
 
 AC_DEFINE_UNQUOTED(QMAILINJECT,"$qmailinject")
 
@@ -115,10 +106,6 @@
             AC_MSG_ERROR([Unable to find your qmail-newmrh file, specify --enable-qmail-newmrh.])
         fi
     ] )
-if test ! -f "$qmailnewmrh"
-then
-    AC_MSG_ERROR([Unable to find your qmail-newmrh file, specify --enable-qmail-qmail-newmrh=/full/path/to/qmail-newmrh])
-fi
 
 AC_DEFINE_UNQUOTED(QMAILNEWMRH,"$qmailnewmrh")
 
