diff -urN vpopmail-5.3.3.orig/vmysql.h vpopmail-5.3.3/vmysql.h
--- vpopmail-5.3.3.orig/vmysql.h	2004-01-11 22:51:29.024989008 +0100
+++ vpopmail-5.3.3/vmysql.h	2004-01-11 22:52:31.456497976 +0100
@@ -51,11 +51,11 @@
 primary key (pw_name, pw_domain ) "
 #else
 #define TABLE_LAYOUT "pw_name char(32) not null, \
-pw_domain char(64) not null, 
+pw_domain char(64) not null, \
 pw_passwd char(40), \
 pw_uid int, pw_gid int, \
 pw_gecos char(48), \
-pw_dir char(160), 
+pw_dir char(160), \
 pw_shell char(20), \
 primary key (pw_name, pw_domain ) "
 #endif
