--- vpopmail-5.4.0-rc1/configure.in.orig	Fri Jan  2 07:08:25 2004
+++ vpopmail-5.4.0-rc1/configure.in	Thu Jan 15 12:04:20 2004
@@ -88,16 +88,6 @@
 
 #----------------------------------------------------------------------
 
-if test `$idcommand` != "0"
-then
-  AC_MSG_WARN(=== vpopmail must be configured as root. ===)
-  AC_MSG_WARN(=== please switch to the root user and  ===)
-  AC_MSG_WARN(=== run ./configure. Thanks ===)
-  AC_MSG_ERROR(aborted.)
-fi
-
-#----------------------------------------------------------------------
-
 if test "$GCC" = yes ; then
   CXXFLAGS="$CXXFLAGS -Wall"
   CFLAGS="$CFLAGS -Wall"
@@ -211,17 +201,6 @@
 
 #----------------------------------------------------------------------
 
-if test ! -d $vpopmaildir/etc
-then
-  mkdir $vpopmaildir/etc
-  chown $vpopmailuid "$vpopmaildir"/etc
-  chgrp $vpopmailgid "$vpopmaildir"/etc
-  chmod 755 "$vpopmaildir"/etc
-  AC_MSG_WARN( making a vpopmail etc directory )
-fi
-
-#----------------------------------------------------------------------
-
 AC_MSG_CHECKING(whether we can locate the qmail directory)
 qmaildir=""
 for f in /var/qmail
@@ -385,89 +364,15 @@
 AC_DEFINE_UNQUOTED(VPOPGROUP,"$vpopgroup","")
 AC_SUBST(vpopgroup)
 
-AC_MSG_RESULT(yes)
-
-AC_MSG_CHECKING(whether password file entry for the vpopmail user exists)
-AC_TRY_RUN( [
-  #include <stdio.h>
-  #include <pwd.h>
-  #include <stdlib.h>
-
-  int main() {
-    struct passwd *pw;
-    char tmpbuf[100];
-    FILE *f;
-
-        unlink("vpopmail.dir");
-        unlink("vpopmail.uid");
-        unlink("vpopmail.gid");
-
-        if ( (f=fopen("vpopusername","r+"))==NULL) {
-                printf("No vpopmail user found.\n");
-                printf("Please add the vchkpw group and\n");
-                printf("vpopmail user. Then run configure again\n");
-                return(-1);
-        }
-
-        fgets(tmpbuf,100,f);
-        fclose(f);
-        tmpbuf[strlen(tmpbuf)-1] = 0;
-
-        if (( pw=getpwnam(tmpbuf)) != 0 ) {
-                f=fopen("vpopmail.dir","a+");
-                fprintf(f,"%s\n", pw->pw_dir);
-                fclose(f);
-
-                f=fopen("vpopmail.uid", "a+");
-                fprintf(f,"%d\n", pw->pw_uid);
-                fclose(f);
-
-                f=fopen("vpopmail.gid", "a+");
-                fprintf(f,"%d\n", pw->pw_gid);
-                fclose(f);
-        } else {
-                printf("No vpopmail user found.\n");
-                printf("Please add the vchkpw group and\n");
-                printf("vpopmail user. Then run configure again\n");
-                return(-1);
-        }
-        return(0);
-  }
-  ],
-    AC_MSG_RESULT(yes),
-    AC_MSG_RESULT(no)
-    AC_MSG_ERROR(Could not compile and run even a trivial ANSI C program - check CC.),
-    AC_MSG_ERROR(Could not compile and run even a trivial ANSI C program - check CC.)
-)
-
-if test ! -f vpopmail.uid
-then
-        AC_MSG_ERROR(No vpopmail user.)
-fi
-
-if test ! -f vpopmail.gid
-then
-        AC_MSG_ERROR(No vpopmail group.)
-fi
-
-if test ! -f vpopmail.dir
-then
-        AC_MSG_ERROR(No vpopmail home directory)
-fi
-
-vpopmaildir=`cat vpopmail.dir`
+vpopmaildir="/var/lib/vpopmail"
 AC_SUBST(vpopmaildir)
 AC_DEFINE_UNQUOTED(VPOPMAILDIR,"$vpopmaildir","")
-rm -f vpopmail.dir
 
-vpopmailuid=`cat vpopmail.uid`
+vpopmailuid="121"
 AC_DEFINE_UNQUOTED(VPOPMAILUID,$vpopmailuid,"")
-rm -f vpopmail.uid
 
-vpopmailgid=`cat vpopmail.gid`
+vpopmailgid="121"
 AC_DEFINE_UNQUOTED(VPOPMAILGID,$vpopmailgid,"")
-rm -f vpopmail.gid
-rm -f vpopusername
 
 #----------------------------------------------------------------------
 
@@ -536,10 +441,10 @@
             if test "$tcpserver_file" = ""
             then
                 AC_MSG_WARN([Unable to find your tcpserver relay file.])
-                AC_MSG_WARN([Creating tcp.smtp in "$vpopmaildir"/etc/tcp.smtp.])
+                AC_MSG_WARN([Creating tcp.smtp in ./tcp.smtp.])
 
-                echo "127.:allow,RELAYCLIENT=\"\"" > "$vpopmaildir"/etc/tcp.smtp
-                tcpserver_file="$vpopmaildir"/etc/tcp.smtp
+                echo "127.:allow,RELAYCLIENT=\"\"" > ./tcp.smtp
+                tcpserver_file="./tcp.smtp"
             fi
         ]
     )
